packages/opc-timeline/demo/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>opc-timeline</title>
+    <script src="opc-timeline.js"></script>
+    <style>
+        div {
+            height: 100px;
+            width: 800px;
+            padding-top: 50px;
+            margin: auto;
+            margin-bottom: 5rem;
+        }
+        .dark {
+            background: #000000;
+        }
+    </style>
+</head>
+<body>
+    <div class="light">
+        <opc-timeline id="timeline0" current-step-index="0">
+        </opc-timeline>
+    </div>
+    <div class="light">
+        <opc-timeline id="timeline1" current-step-index="1">
+            <span slot="start-label" style="font-weight: 600;">Loaned on: Apr 28, 2018</span>
+            <span slot="end-label" style="font-weight: 600;">Expires on: Oct 27, 2018</span>
+        </opc-timeline>
+    </div>
+    <div class="dark">
+        <opc-timeline id="timeline2" current-step-index="2">
+            <span slot="start-label" style="font-weight: 600; color: white;">Loaned on: Apr 28, 2018</span>
+            <span slot="end-label" style="font-weight: 600; color: white;">Expires on: Oct 27, 2018</span>
+        </opc-timeline>
+    </div>
+    <div class="dark">
+        <opc-timeline id="timeline3" current-step-index="3">
+        </opc-timeline>
+    </div>
+    <script>
+        document.querySelector('#timeline0').steps = [undefined, 'Scheduled', 'Loaned', 'Extended', 'Completed', null];
```

Comment:
Why there's undefined and null?
```js
        [undefined, 'Scheduled', 'Loaned', 'Extended', 'Completed', null]
```

Reply:
To represent empty states

<img width="909" alt="Screenshot 2020-10-06 at 12 14 34 AM" src="https://user-images.githubusercontent.com/12994292/95119012-ec74c180-0768-11eb-805d-cdd4bac8bfe9.png">